hieradata/site/ls/role/ipamaster.yaml

```diff
@@ -0,0 +1,2 @@
+---
+easy_ipa::ipa_master_fqdn: "ipa1.cp.lsst.org"
```

Comment:
Should this be `ipa1.ls.lsst.org`?

Reply:
No, this is the IPA master that the replica attaches to.